csi/src/meta_data.rs

```diff
@@ -1078,28 +1154,35 @@ impl MetaData {
                 util::format_anyhow_error(&e),
             ),
         };
-        let mount_res = util::mount_volume_bind_path(
-            &vol_path,
-            &target_path,
-            bind_mount_mode,
-            mount_options,
-            fs_type,
-            read_only,
-        )
+        let vol_path_owned = vol_path.to_owned();
+        let target_path_owned = target_path.to_owned();
+        let fs_type_owned = fs_type.to_owned();
+        let mount_options_owned = mount_options.to_owned();
+        let mount_res = smol::unblock(move || {
+            util::mount_volume_bind_path(
+                &vol_path_owned,
+                &target_path_owned,
+                bind_mount_mode,
+                &mount_options_owned,
+                &fs_type_owned,
+                read_only,
+            )
+        })
+        .await
         .context(format!(
             "failed to bind mount from {:?} to {:?}",
             vol_path, target_path,
         ));
         if let Err(bind_err) = mount_res {
             if ephemeral {
-                match self.delete_volume_meta_data(vol_id) {
+                match self.delete_volume_meta_data(vol_id).await {
                     Ok(_) => debug!(
                         "successfully deleted ephemeral volume ID={}, when bind mount failed",
                         vol_id,
                     ),
                     Err(e) => error!(
                         "failed to delete ephemeral volume ID={}, \
-                            when bind mount failed, the error is: {}",
+                        when bind mount failed, the error is: {}",
```

Comment:
extra 4 spaces indentation before the second text line

Reply:
Done